### ISSUE TITLE
[Bug][Grid] Missing Alt/Option(Mac) + Arrow Right/Left key combination for resizing a column

### ISSUE THREAD

**Issue by silviyaboteva** [telerik/kendo-react#3070]
### Description

The official specification of the Grid keyboard navigation lists a key combination for resizing the column via `Alt/Option(Mac) + Arrow Right/Left`. Ref: https://www.telerik.com/design-system/docs/components/grid/accessibility/

However, this feature is missing in our KendoReact Grid.

Reported in Ticket ID: 1684138

### Plan

KendoReact Free

### Steps To Reproduce

1. Go to https://www.telerik.com/kendo-react-ui/components/grid/accessibility/keyboard-navigation
2. Navigate to a column header cell
3. Try `Alt/Option(Mac) + Arrow Right/Left` combination

### Actual Behavior

The focus is moved to the next cell.

### Expected Behavior

To resize the column.

### Browser

All

### Browser version

latest

### OS type

MacOS
